packages/client/src/components/AppNavBar/AppNavBar.tsx

```diff
@@ -91,11 +145,7 @@ const AppNavBar = () => {
             open={Boolean(menuAnchorElem)}
             onClose={handleMenuClosed}
           >
-            {PAGES.map((page) => (
-              <MenuItem key={page.pathname} component={Link} href={page.pathname}>
-                {page.name}
-              </MenuItem>
-            ))}
+            {auth.isLoggedIn ? menuItems(LEFT_PAGES) : menuItems(PAGES)}
```

Comment:
Można uniknąć powtórzenia `menuItems` (podobnie z prawą sekcją navbara poniżej):
```suggestion
            {menuItems(auth.isLoggedIn ? LEFT_PAGES : PAGES)}
```